packages/@okta/vuepress-site/docs/guides/configure-access-policy/limit-scopes-clients-can-access/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Limit which scopes some clients can access
+---
+
+Use the following steps to create a policy that limits which scopes that designated clients can access.
+
+## Create the policy container
+
+1. In the Developer Console, navigate to **API > Authorization Servers**.
+
+2. Select the name of the authorization server that you want to apply the policy to.
+
+3. Select **Access Policies**, and then **Add Policy**.
+
+4. Enter a **Name** and a **Description** for the policy.
+
+5. Select **The following clients:** and start typing the names of the Okta OpenID Connect applications that you want to cover with the access policy. This field automatically displays a list of applications that match what you type.
+
+6. Click **Create Policy**.
+
+## Create the rule
+
+1. Select the name of the access policy that you just created, and then select **Add Rule**.
+
+2. Enter a **Name** for the rule, and then for this example leave the defaults for **IF Grant type is** and **AND User is**.
+
+3. For the **AND Scopes requested** field, select **The following scopes:** and then click **OIDC default scopes**. All of the OpenID Connect reserved scopes appear in the box. 
+
+4. Click the **x** to remove all of the scopes except for **openid** and **email**. This limits the scopes that the applications associated with this policy can request to just those two scopes. 
+
+> **Note:** Access policies rules are whitelists. Verify that there are no other rules in the policy that have **any scopes** selected and that all of your rules match only the `openid` and/or the `email` scopes.
```

Comment:
We describe the scopes with **bold** above and `code` in this note. Is there a way to keep the formatting consistent?

Reply:
The bolding in step 4 is because they are removing items in the UI. Our standard is to bold UI elements that the user is interacting with. General reference to just the scope is `openid` and `email`.